### DISCUSSION TITLE
Lazer on arm based windows devices.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20364] [Ideas] [answered]
Hello , osu! lazer developers .
I now have a arm64 and arm32 based windows devices , could them have osu lazer?
I want to see not only Apple arm64 builds but also windows arm(64) builds in the release of osu! lazer!
Thanks for bringing osu! lazer to us :)

(I have searched for "arm","ARM","Windows ARM",but got no results)

**Comment by Theighlin** [comment, marked as answer]
https://github.com/ppy/osu/issues?q=is%3Aissue+is%3Aopen+arm+windows

**Reply by ghost** [reply]
Thank you!
I'm very sorry for not searching in issues but only discussions.
